contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -843,10 +843,10 @@ having the `+optional` comment tag is highly recommended.
 Required fields have the opposite properties, namely:
 
 - They do not have an `+optional` comment tag.
-- They do not have an `omitempty` struct tag.
-- They are not a pointer type in the Go definition (e.g. `AnotherFlag SomeFlag`).
+- They mark themselves as required explicitly with a `+required` comment tag, or implicitly by not setting an `omitempty` struct tag.
```

Comment:
I don't have enough context, but could we simplify this guidance to required fields are only marked via the +required marker (given that's how it should be)